airbyte_cdk/sources/declarative/interpolation/filters.py

```diff
@@ -135,5 +136,51 @@ def regex_search(value: str, regex: str) -> str:
     return ""
 
 
-_filters_list = [hash, base64encode, base64decode, base64binascii_decode, string, regex_search]
+def hmac(value: Any, key: str, hash_type: str = "sha256") -> str:
+    """
+    Implementation of a custom Jinja2 hmac filter with SHA-256 support.
+
+    This filter creates a Hash-based Message Authentication Code (HMAC) using a cryptographic
+    hash function and a secret key. Currently only supports SHA-256, and returns hexdigest of the signature.
+
+    Example usage in a low code connector:
+
+    auth_headers:
+      $ref: "#/definitions/base_auth"
+      $parameters:
+        signature: "{{ 'message_to_sign' | hmac('my_secret_key') }}"
```

Comment:
Ah gotcha, so to use this in many cases its up to the caller to create the list of query params by hand to sign.

Think thats a fine work around.

But it seems like the next win is
1. a jinja function to get all query params
2. a jinja function to transform it into a delimited string (after you optionally sort / filter / map the list)

Reply:
Yes, that would indeed make things easier.